plotting/templates/rSUBSETtemplate.txt

```diff
@@ -0,0 +1,17 @@
+library(jsonlite)
+
+query <- '{q}'
+queryObj <- fromJSON(query)
+fname <- paste("script_output",".nc")
```

Comment:
`paste` concatenates strings. Since we're not adding anything to the output filename you can just set
`fname <- "script_output.nc"`